calico_node/filesystem/etc/rc.local

```diff
@@ -8,37 +8,50 @@ if [ "$CALICO_DISABLE_FILE_LOGGING" == "true" ]; then
 	rm -r /etc/service/bird6/log
 	rm -r /etc/service/confd/log
 	rm -r /etc/service/felix/log
+	rm -r /etc/service/calico-bgp-daemon/log
```

Comment:
Now that we're adding multiple backends, I would prefer that instead of removing the services we don't need, we "add" the ones we do. 

The current architecture places all service files in calico-node in `/etc/service`, then [the "start_runit" script runs `exec /sbin/runsvdir -P /etc/service`](https://github.com/projectcalico/calico-containers/blob/master/calico_node/start_runit#L16). I think a better solution would be to push the necessary services from an "available" folder to an "enabled" folder (kind of like apache HTTP does).

To do this, we'd need to make the following changes:
1. Move our service source location in this repository into a new "available" subdirectory: `mv calico_node/filesystem/etc/service/ calico_node/filesystem/etc/service/available`
2. Modify `rc.local` so that it creates an empty folder at `/etc/service/enabled/`
3. Modify `rc.local` so that for each branch statement, it moves its necessary files from "available" into "enabled", e.g.:`mv /etc/service/available/bird /etc/service/enabled/bird`
4. Modify `start_runit` so that it loads `/etc/service/enabled/*`